src/game/client/input/keyboard.cpp

```diff
@@ -284,7 +294,7 @@ void Keyboard::Update_Keys()
  */
 wchar_t Keyboard::Translate_Key(wchar_t key)
 {
-    if (key > 256) {
+    if (key > 255) {
```

Comment:
@duncanspumpkin is right, use the constant we already defined, `>= 256` is the same as `> 255` logically.